ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -486,3 +486,9 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None, aweights=N
     return ivy.cov(
         m, y, rowVar=rowvar, bias=bias, ddof=ddof, fweights=fweights, aweights=aweights
     )
+
+# percentile
+
+@to_ivy_arrays_and_back
+def percentile(a, q, axis=None, out=None, overwrite_input=False, interpolation="linear", keepdims=False):
```

Comment:
The function signature doesn't match with jax's documentation of percentile (https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.percentile.html)